adapters/adtrgtme/adtrgtme.go

```diff
@@ -27,7 +27,7 @@ func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server co
 
 func (v *adapter) MakeRequests(
 	openRTBRequest *openrtb2.BidRequest,
-	requestInfo *adapters.ExtraRequestInfo,
+	_ *adapters.ExtraRequestInfo,
```

Comment:
Same comment here. Let's pass in the name instead of `_` to be consistent with other adapters.